tests/backup_helper.go

```diff
@@ -6357,6 +6398,72 @@ func GetAllVMsInNamespacesWithLabel(namespaceLabel map[string]string) ([]kubevir
 
 }
 
+// GetAllVMsFromScheduledContexts returns all the Kubevirt VMs in the scheduled contexts
+func GetAllVMsFromScheduledContexts(scheduledContexts []*scheduler.Context) ([]kubevirtv1.VirtualMachine, error) {
```

Comment:
How about we define a method on *scheduler.Context?

Reply:
All I need is the namespace name from the context here

Reply:
If we define a method on *scheduler.Context, we can call appCtx.GetVMs().